src/index.html

```diff
@@ -22,6 +22,357 @@
+            <a
+              href="#"
+              class="header__link header__link--is-active is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__info">
+          <img
+            class="product-card__image"
+            src="images/imac.jpeg"
+            alt="product"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">Product code: 195434</p>
+        </div>
+        <div class="product-card__reviews">
+          <div class="product-card__reviews-stars">
+            <div class="product-card__star product-card__star--1--filled"></div>
+            <div class="product-card__star product-card__star--2--filled"></div>
+            <div class="product-card__star product-card__star--3--filled"></div>
+            <div class="product-card__star product-card__star--4--filled"></div>
+            <div class="product-card__star product-card__star--5"></div>
+          </div>
+          <div class="product-card__reviews-total">Reviews: 5</div>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price_title">Price:</p>
+          <p class="product-card__price_total">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="product-card__buy-button"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute is used for testing purposes and should not be present in the production code unless it is being used for automated tests.